frontend/src/app/shared/pipes/paragraph.pipe.ts

```diff
@@ -1,22 +1,21 @@
 import { Pipe, PipeTransform } from '@angular/core';
 
-
 @Pipe({
     name: 'paragraph',
 })
 export class ParagraphPipe implements PipeTransform {
 
     transform(content: string | string[]): unknown {
-        const splitText = this.addParagraphTags(content);
+        const splitText = this.addParagraphBreaks(content);
         return splitText;
     }
 
-    addParagraphTags(content: string | string[]) {
+    addParagraphBreaks(content: string | string[]) {
         const paragraphs = typeof content === 'string' ? content.split('\n') : content;
         if (!paragraphs || paragraphs.length === 1) {
             return content;
         }
-        return paragraphs.map(p => `<p>${p}</p>`).join(' ');
+        return paragraphs.filter(p => p !== '').join('<br><br>');
```

Comment:
Paragraphs `<p>` and linebreaks `<br>` are different things; linebreaks are used for anything that isn't a paragraph break, e.g. in postal addresses or poems. This concerns actual paragraphs in a text, though, so it's not appropriate to use `<br>` here.

Using paragraphs appropriately makes the text much more user-friendly for screen readers, especially if you're close-reading a longer document. See also [accessibility concerns for `<br>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br#accessibility).